python/lsst/ts/externalscripts/auxtel/latiss_cwfs_align.py

```diff
@@ -667,8 +748,22 @@ def set_metadata(self, metadata):
         metadata.filter = f"{self.filter},{self.grating}"
 
     async def arun(self, checkpoint=False):
-        """Perform the loop to perform CWFS measurements and hexapod
-        adjustments until the threshold is reached."""
+        """Perform CWFS measurements and hexapod adjustments until the
+        threshold is reached.
+        """
+
+        if self.cwfs_target is not None and self.cwfs_target_dec is None:
+            await self.slew_object(
+                name=self.cwfs_target, rot=0.0, rot_type=RotType.PhysicalSky
+            )
+        elif self.cwfs_target is not None and self.cwfs_target_dec is not None:
```

Comment:
If there is no ra specified, does slew_icrs give a coherent error message saying so?

Reply:
I think I see your point... Here I am assuming the script was configured using the configure method which will never yield dec without ra. But I imagine one could try to run it without going over configure. I am actually not sure what would happen in it ra is None, so I'll protect against it.